docs/book/how-to/develop-locally/keep-your-pipeline-dashboard-clean.md

```diff
@@ -0,0 +1,35 @@
+---
+description: Learn how to keep your pipeline runs clean during development.
+---
+
+# Keep your pipelines and dashboard clean
+
+When developing pipelines, it's common to run and debug them multiple times. To avoid cluttering the server with these development runs, ZenML provides several options:
+
+## Unlisted Runs
+
+Pipeline runs can be created without being explicitly associated with a pipeline by passing the `unlisted` parameter when running a pipeline:
+
+```python
+pipeline_instance.run(unlisted=True)
+```
+
+Unlisted runs are not displayed on the pipeline's page in the dashboard, keeping the pipeline's history clean and focused on important runs.
+
+## Deleting Pipelines
```

Comment:
how about deleting pipeline runs? mihgt be worth adding a script to delete the last 24 hours of pipeline runs?

Reply:
added